apps/userdashboard/serializers.py

```diff
@@ -21,6 +22,7 @@ class ModerationCommentSerializer(serializers.ModelSerializer):
     user_name = serializers.SerializerMethodField()
     user_image = serializers.SerializerMethodField()
     user_profile_url = serializers.SerializerMethodField()
+    ai_report = AiReportSerializer(read_only=True)
```

Comment:
@philli-m here is the serializer, it returns all fields of the AIReport. Do you want to rebase your [PR](https://github.com/liqd/a4-defakts/pull/17#pullrequestreview-1578982426) after we merge this tomorrow? Then you can create an ai_report with the python shell and saved in the DB for testing it. . Let me know during standup.